darts/models/forecasting/tft_model.py

```diff
@@ -637,6 +668,7 @@ def __init__(
         add_relative_index: bool = False,
         loss_fn: Optional[nn.Module] = None,
         likelihood: Optional[Likelihood] = None,
+        norm_type: str = "LayerNorm",
```

Comment:
how about renaming to `layer_norm_variant` ? To make it clear it touches the layer norm part of the transformer.